### PR TITLE
Teamwork Makes the Dreamwork BABY

### PR DESCRIPTION
**Pull request by nathanroberts55**
Implemented many things.

<details>

Desktop for Challenge

Mobile for Challenge

</details>

### CODE REVIEW

File: src/routes/settings/[id]/+page.svelte [hunk 1]
```diff
@@ -17,36 +17,49 @@
...
-			</a>
-			<button class="py-2 px-4 rounded-lg bg-blue-400 text-white max-md:text-2xl text-xl">Start Challenge</button>
-		</div>
+		<form method="GET" action="/new" class="flex flex-col justify-between h-full">
+			<div class="space-y-4 px-4">
+				<Toggle label="Weapons Challenge" name="weapons" />
+				<Toggle label="Time Challenge" name="time" />
+				<!-- <Toggle label="Traversal Challenge" name="traversal" /> -->
+				<Toggle label="Kill Challenge" name="kills" />
+				<!-- <Toggle label="Disguise Challenge" name="disguise" /> -->
```
Comment (dylhack) [inline src/routes/settings/[id]/+page.svelte]: are these commented out toggles going to be utilized?

File: src/routes/new/+page.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+			mapName,
+			weaponsVal,
+			timeVal,
+			traversalVal,
+			killsVal,
+			disguiseVal,
+			customVal,
+			seed
+		})
+	);
```
Comment (dylhack) [inline src/routes/new/+page.js:27]: move encoding and decoding to a util module

File: src/routes/challenge/[challenge]/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+			<div class="flex flex-col space-y-2 text-white">
+				<!-- <p class="font-bold text-center">Difficulty: {difficulty}</p> -->
+
+				<!-- <List name="Targets" items={map.targets} />
+				{#if data.killsVal == true}
+					<List name="Challenge" items={challenges} />
+				{/if}
+				{#if data.weaponsVal == true}
+					<List name="Weapons" items={weapons} />
+				{/if} -->
```
Comment (dylhack) [inline src/routes/challenge/[challenge]/+page.svelte]: can this get deleted?

**Review by dylhack — CHANGES_REQUESTED** [review]

File: src/routes/challenge/[challenge]/+page.svelte (hunk 3 above)
Comment (dSolid) [reply src/routes/challenge/[challenge]/+page.svelte]: this should totally be deleted :)

File: static/hitman.jpg
Comment (dSolid) [inline static/hitman.jpg:1]: this should be commited using git lfs instead https://git-lfs.com/

File: static/maps/ambrose-island.jpg
Comment (dSolid) [inline static/maps/ambrose-island.jpg:1]: same here even though the file is smaller

File: src/routes/settings/[id]/+page.svelte (hunk 1 above)
Comment (nathanroberts55) [reply src/routes/settings/[id]/+page.svelte]: Done

File: src/routes/new/+page.js (hunk 2 above)
Comment (nathanroberts55) [reply src/routes/new/+page.js:27]: No

File: src/routes/challenge/[challenge]/+page.svelte (hunk 3 above)
Comment (nathanroberts55) [reply src/routes/challenge/[challenge]/+page.svelte]: Deleted

File: static/hitman.jpg
Comment (nathanroberts55) [reply static/hitman.jpg:1]: Tell that to vscode buddy

File: static/maps/ambrose-island.jpg
Comment (nathanroberts55) [reply static/maps/ambrose-island.jpg:1]: mmmm sounds like a @dylhack problem to me

**Review by dylhack — APPROVED** [review]

**Pull request merged**